app/views/projects/samples/show.html.erb

```diff
@@ -51,6 +51,19 @@
         },
         class: "button button--size-default button--state-default action-link" %>
       <% end %>
+      <% header.with_action do %>
+        <%= link_to t(".delete_files_button"),
+        new_namespace_project_sample_attachments_deletion_path(
+          sample_id: @sample.id
+        ),
+        data: {
+          turbo_frame: "sample_files_modal",
+          turbo_stream: false,
+          controller: "action-link",
+          action_link_required_value: 1
+        },
+        class: "button button--size-default button--state-default action-link" %>
```

Comment:
Can we update the class of this button to `button--state-destructive` from `button--state-default`. This way the button sticks out instead of blending in with the rest

Reply:
c99b4e8a879e3e2703491cad6738cfc273014707